/left_topic_name: '/left_image'
/right_topic_name: '/right_image'
/absolute_path: false
/calibrate: true
/save_path: 'calib.yml'
/absolute_video_path: true
/left_video_path: '/media/chao/RAID1_L/chaoz/video_recordings/stereo/06_l.avi'
/right_video_path: '/media/chao/RAID1_L/chaoz/video_recordings/stereo/06_r.avi'
